cloud_governance/policy/common_policies/send_aggregated_alerts.py

```diff
@@ -141,7 +141,7 @@ def __update_delete_days(self, policy_es_data: list):
                     days = int(days)
                 if not days:
                     days = 0
-                alert_user = False
+                alert_user = True
```

Comment:
Why it should true by default ?

Reply:
Changed based on the env variable ALERT_DRY_RUN.